projects/packages/waf/src/class-waf-runner.php

```diff
@@ -45,7 +48,7 @@ public static function initialize() {
 		if ( function_exists( 'add_action' ) ) {
 			self::add_hooks();
 		}
-		if ( ! self::did_run() ) {
+		if ( ! self::did_run() && self::rules_enabled() ) {
```

Comment:
Idea here being that if there's no rules enabled, there's no need to run the firewall.

Reply:
This has been removed to prevent unnecessary overhead on each request :+1: